### PR TITLE
Refresh build pipeline

### PR DESCRIPTION
**Pull request by twometresteve**
## Goal

Ensure the existing Buildkite pipeline runs.

## Changeset

Just a trivial change of the queue name.

## Testing

Covered by CI.

### CODE REVIEW

File: package.json
Comment (tomlongridge) [inline package.json:1]: Is this file really required?

Comment (twometresteve) [reply package.json:1]: Nope, just a fail of mine, now removed.

**Review by tomlongridge — APPROVED** [review]
